sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -129,10 +129,13 @@ class PerElement(ptransform.PTransform):
 
     def expand(self, pcoll):
       paired_with_void_type = KV[pcoll.element_type, Any]
-      return (pcoll
-              | ('%s:PairWithVoid' % self.label >> core.Map(lambda x: (x, None))
-                 .with_output_types(paired_with_void_type))
-              | core.CombinePerKey(CountCombineFn()))
+      output_type = KV[pcoll.element_type, int]
```

Comment:
Is this type update from Any to int needed for the test suite ?

cc: @robertwb

Reply:
The output type needs to be as narrow as possible in order to avoid python pickled coder. The test was failed because of it.